themes/cardinal_service/lib/scss/opportunities/index.scss

```diff
@@ -0,0 +1,7 @@
+@charset 'UTF-8';
+
+// Decanter
+@import '../config';
```

Comment:
config should already be imported at this point

Reply:
Awesome! I've removed it from both Spotlight and Opportunities respective `index.scss` files